src/integrationTest/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/component/testutils/IntegrationConstants.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.component.testutils;
+
+public class IntegrationConstants {
+
+    public static final String HOME_OFFICE_API_URL = "/ichallenge/applicationStatus/getBySearchParameters";
+
+    public static final String SERVICE_TOKEN = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwiaW"
+                                               + "F0IjoxNTE2MjM5MDIyfQ.L8i6g3PfcHlioHCCPURC9pmXT7gdJpx3kOoyAfNUwCc";
+
+    public static final String USER_ID = "49154ae9-47be-4469-9edd-d43f68d245f0";
+
+    public static final String SYSTEM_USER_TOKEN = "eyJ0eXAiOiJKV1QiLCJ6aXAiOiJOT05FIiwia2lkIjoiMWVyMFdSd2dJT1RBRm9q"
```

Comment:
We are probably not using a SYSTEM_USER (as in timed-event-service) for HO.  Just a regular user.

Reply:
Done